tests/unit/tabular/test_base.py

```diff
@@ -343,3 +344,38 @@ def test_sample_batches_transform_conditions_correctly():
     model._sample_batch.assert_any_call(
         1, 100, 10, {'column1': 30}, {'transformed_column': 70}, 0.01
     )
+
+
+def test__make_conditions_df_without_num_rows():
```

Comment:
Let's also add a test for when `num_rows` is not `None`. We can pass in `num_rows = 100` (or some value different from `self._num_rows`), so that we know `_make_conditions_df` also works in this case.